bucket/houdoku.json

```diff
@@ -0,0 +1,28 @@
+{
+    "version": "2.9.2",
+    "description": "Manga reader and library manager for the desktop",
+    "homepage": "https://houdoku.org/",
+    "license": "MIT",
+    "url": "https://github.com/xgi/houdoku/releases/download/v2.9.2/Houdoku-Setup-2.9.2.exe#/dl.7z",
+    "hash": "sha512:db535de9ac5de28a5d4e7639591be440e166542aa07066829dc7c81f3e0bbb4e3da54a31b031b3a262935ceb67d355da6d32d44e6eb926cb2d5cd10eb64e733c",
```

Comment:
Houdoku is 64-bit so it should be wrapped inside an `architecture` block. Fixed in https://github.com/ScoopInstaller/Extras/pull/9868